test/test_nn.py

```diff
@@ -5289,6 +5289,14 @@ def test_ConvTranspose3d_correct_output_size(self):
         i = torch.rand(1, 2, 1, 1, 1)
         out = m(i, output_size=(1, 2, 2, 2, 2))
 
+    @unittest.skipIf(not TEST_CUDA, 'CUDA not available')
+    def test_ConvTranspose2d_half_cublas_gemm(self):
+        inputs = torch.randn(1, 1, 16, 16, device='cuda', dtype=torch.half)
```

Comment:
you can also force CuDNN to be off (for future safety) using the context manager we have available (i think it's called with torch.backends.cudnn something something

Reply:
Thanks! I've added the context manager.